cmd/renterd/main.go

```diff
@@ -179,6 +179,7 @@ func main() {
 	flag.DurationVar(&workerCfg.SessionTTL, "worker.sessionTTL", 2*time.Minute, "the time a host session is valid for before reconnecting")
 	flag.DurationVar(&workerCfg.DownloadSectorTimeout, "worker.downloadSectorTimeout", 3*time.Second, "timeout applied to sector downloads when downloading a slab")
 	flag.DurationVar(&workerCfg.UploadSectorTimeout, "worker.uploadSectorTimeout", 5*time.Second, "timeout applied to sector uploads when uploading a slab")
+	flag.IntVar(&workerCfg.UploadMaxOverdrive, "worker.uploadMaxOverdrive", 5, "maximum number of overdrive workers used when uploading a slab")
```

Comment:
```suggestion
	flag.IntVar(&workerCfg.UploadMaxOverdrive, "worker.uploadMaxOverdrive", 5, "maximum number of overdrive workers when uploading a slab")
```

Reply:
bit of a `nit` but I think this way it's more clear we are talking about max overdrive workers at any given time rather than cap on the total overdrive workers used to upload the slab